_data/documentation.yml

```diff
@@ -0,0 +1,46 @@
+
+
+General:
+  Project-wide Documentation: https://jupyter.readthedocs.io
+  Contributing to Jupyter: https://jupyter.readthedocs.io/en/latest/contributor/content-contributor.html
+
+User Interfaces:
+  JupyterLab: https://jupyterlab-tutorial.readthedocs.io/en/latest/
+  Jupyter Notebook: https://jupyter-notebook.readthedocs.io/en/stable/
+  Jupyter Console: https://jupyter-console.readthedocs.io/en/stable/
+  Qt Console : https://qtconsole.readthedocs.io/en/stable/
+
+IPython:
+  IPython: https://ipython.readthedocs.io/en/stable/
+  IPyParallel: https://ipyparallel.readthedocs.io/en/stable/
+  Traitlets: https://traitlets.readthedocs.io/en/stable/
+
+Kernels:
+  IPython: https://ipython.readthedocs.io/en/stable/
+  IRkernel: https://irkernel.github.io/
+  IJulia: https://github.com/JuliaLang/IJulia.jl
+  Community maintained kernels: https://github.com/jupyter/jupyter/wiki/Jupyter-kernels
+
+Widgets:
+  IPyWidgets: https://ipywidgets.readthedocs.io/en/latest/
```

Comment:
JupyterWidgets??

Reply:
Depends if you speak of the Js library or the Python bindings !